apps/events/tests/test_timeout_trigger.py

```diff
@@ -275,15 +275,15 @@ def test_new_human_message_resets_count(session):
 @pytest.mark.django_db()
 @pytest.mark.parametrize(
     ("status", "matches"),
-    {
```

Comment:
This change was required since the ordering of the set can change between workers. xdist requires the exact same test suite to be generated between workers so it can choose which ones to pick.